charts/agent/values.yaml

```diff
@@ -280,6 +280,31 @@ config:
         - action: insert
           key: loki.resource.labels
           value: project, license, deployment_environment, service, namespace, container, aws_region
+    resource/loki-raw:
+      attributes:
+        - action: insert
+          key: loki.format
+          value: raw
+        - action: insert
+          key: loki.resource.labels
+          value: project, license, deployment_environment, service, namespace, container,
```

Comment:
`        - action: insert
          key: loki.resource.labels
          value: project, license, deployment_environment, service, namespace, container,`

this part must be common  without copy. Try to create "resource/loki" - for both pipes - it would be more applicable

Reply:
Done